clients/python/girder_client.py

```diff
@@ -434,7 +437,7 @@ def _transformFilename(self, name):
         name = name.replace(os.path.sep, ' ')
         if os.path.altsep:
             name = name.replace(os.path.altsep, ' ')
-        return re.sub(r'^[/\\]+', '', name)
+        return _safeNameRegex.sub('', name)
```

Comment:
Do we really want to substitute the empty string (I realize that you didn't just add that)?  What if the name is just the restricted characters?  In the lines above, we substituted a space.